tools/ports/mpg123.py

```diff
@@ -2105,7 +2105,7 @@ def show():
 ,	unsigned char **v1, size_t *v1_size
 ,	unsigned char **v2, size_t *v2_size );
 
-/** Point icy_meta to existing data structure wich may change on any next read/decode function call.
+/** Point icy_meta to existing data structure which may change on any next read/decode function call.
  *  \param mh handle
```

Comment:
Can you revert all the changes to this file?   This content is coming form upstream so they are not really out spelling mistakes to fix and this change could make comparing with upstream harder.